### PR TITLE
Remove empty rows when parsing a csv.

### PR DESCRIPTION
**Pull request by tommyvdv**

### CODE REVIEW

File: spoon/file/csv.php [hunk 1]
```diff
@@ -252,6 +252,12 @@ public static function fileToArray($path, array $columns = array(), array $exclu
 				// remove the original field from this row
 				unset($row[$columnId]);
 			}
+
+            // Remote empty rows.
+            if(count($row) != count(array_filter($row)))
```
Comment (WouterSioen) [inline spoon/file/csv.php]: Indentation isn't correct.

Comment (WouterSioen) [inline spoon/file/csv.php]: Btw, this will also remove the row if only one column is empty.

**Comment by carakas** [comment]
@tommyvdv the indentation of this library is with tabs

**Comment by carakas** [comment]
closed because of inactivity

**Pull request closed without merging**